doc/sphinx/source/hardware/comm_interface.rst

```diff
@@ -6,11 +6,11 @@ Communication Interface
 
    For data transmission to IIO clients, IIO firmware applications uses *Virtual Serial*
    Or *UART* as primary communication links. Firmware by default uses the Virtual Serial
-   interface for higher speed data transmission as SDP-K1 MCU board supports
-   both Virtual Serial and UART interface. If you target a different MCU board that does not 
+   interface for SDP-K1 for higher speed data transmission as MCU board supports
+   both Virtual Serial and UART interface. The Nucleo-H563 supports only UART. If you target a different MCU board that does not 
```

Comment:
I think we now support Vcom for h563, isn't it?

Reply:
I dont think we've released that yet